ui/src/app/core/initial-error/initial-error.component.html

```diff
@@ -1,15 +1,4 @@
 <div *ngIf="initialLoadFailure" class="error-message">
-  <table class="monospace">
-      <tr>      _       _       __  __                                   </tr>
-      <tr>     | |     | |     |  \/  |                                  </tr>
-      <tr>     | | ___ | |__   | \  / | __ _ _ __   __ _  __ _  ___ _ __ </tr>
-      <tr> _   | |/ _ \| '_ \  | |\/| |/ _` | '_ \ / _` |/ _` |/ _ \ '__|</tr>
-      <tr>| |__| | (_) | |_) | | |  | | (_| | | | | (_| | (_| |  __/ |   </tr>
-      <tr> \____/ \___/|_.__/  |_|  |_|\__,_|_| |_|\__,_|\__, |\___|_|   </tr>
-      <tr>                                                __/ |          </tr>
-      <tr>                                               |___/           </tr>
-  </table>
-  <p>
```

Comment:
This broke because, in the new version of Angular, all this leading whitespace in the `tr` elements got trimmed out. Could fix by adding a lot of `nbsp;` but was very fiddly and made this file kind of horrifying.

Reply:
Saddd.